uninstall.php

```diff
@@ -1,51 +1,48 @@
 <?php
-
-// If uninstall not called from WordPress exit
 defined( 'WP_UNINSTALL_PLUGIN' ) or die( 'Cheatin&#8217; uh?' );
 
-// Delete all transients
+// Delete all transients.
 delete_site_transient( 'update_wprocket' );
 delete_site_transient( 'update_wprocket_response' );
 delete_transient( 'wp_rocket_settings' );
 delete_transient( 'rocket_check_licence_30' );
```

Comment:
This has been deleted in 2.9.7. It seems that this branch isn't updated with the latest master content.